docker/build_scripts/build.sh

```diff
@@ -48,11 +48,11 @@ mkdir -p /opt/python
 build_cpythons $CPYTHON_VERSIONS
 rm -rf /usr/local/ssl
 
-# Install patchelf and auditwheel (latest)
-curl -sLO http://nixos.org/releases/patchelf/patchelf-0.9/patchelf-0.9.tar.gz
-tar -xzf patchelf-0.9.tar.gz
-(cd patchelf-0.9 && ./configure && make && make install)
-rm -rf patchelf-0.9.tar.gz patchelf-0.9
+# Install patchelf and auditwheel (latest with unreleased bug fixes)
+curl -sLO http://nipy.bic.berkeley.edu/manylinux/patchelf-0.9njs2.tar.gz
```

Comment:
now that I think about it, it's kinda not-so-great that we're installing this directly over a plain unencrypted `http` link to a random server. It means that if `nipy.bic.berkeley.edu` gets 0wned or MITMed, then some large proportion of the internet gets 0wned too (basically everyone using pip on linux). We can't help having a somewhat broad trust footprint (github, travis, docker hub for our base image, fedora, python.org, pypi for auditwheel, the TLS certificate system), but we should still try to keep it down if we can :-).

Maybe the best strategy would be to check the hash, like we do with openssl?